feat(genapi): update faq location (#6772)
scaleway/docs-content: pages/generative-apis/faq.mdx

For *Dedicated Deployment*, model versions are fixed and provided in the Deployment parameters.

### Where are the inference servers located?

> All *Serverless models* are currently hosted in a secure data center located in Paris, France, operated by [OPCORE](https://www.opcore.com/). This ensures low latency for European users and compliance with European data privacy regulations. In the future, we may add additional location in other European regions and will continue to guarantee *Serverless models* accessible through `api.scaleway.ai` will be hosted in Europe. Customers will be notified when location changes from France-only to Europe.

Reviewer: ```suggestion
All *Serverless models* are currently hosted in a secure data center located in Paris, France, operated by [OPCORE](https://www.opcore.com/). This ensures low latency for European users and compliance with European data privacy regulations. In the future, we may add further locations in other European regions, and we will continue to guarantee that *Serverless models* accessible through `api.scaleway.ai` remain hosted in Europe. Customers will be notified when the hosting location changes from France-only to Europe.
```

> To ensure processing occurs in one single region, we recommend to use *Dedicated Deployment*.

Reviewer: ```suggestion
To ensure that processing stays in a single region, Scaleway recommends using *Dedicated Deployment*.
```
